### PR TITLE
AdaptiveTimeSteppingEbos: rename to AdaptiveTimeStepping

### PR DESCRIPTION
**Pull request by akva2**
And another

### CODE REVIEW

**Comment by akva2** [comment]
jenkins build this please

File: tests/test_RestartSerialization.cpp [hunk 1]
```diff
@@ -112,11 +112,11 @@ TEST_FOR_TYPE(SimulatorReport)
...
-namespace Opm { using ATE = AdaptiveTimeSteppingEbos<Properties::TTag::EbosTypeTag>; }
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosHardcoded, serializationTestObjectHardcoded)
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosPID, serializationTestObjectPID)
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosPIDIt, serializationTestObjectPIDIt)
-TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingEbosSimple, serializationTestObjectSimple)
+namespace Opm { using ATE = AdaptiveTimeStepping<Properties::TTag::EbosTypeTag>; }
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingHardcoded, serializationTestObjectHardcoded)
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingPID, serializationTestObjectPID)
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingPIDIt, serializationTestObjectPIDIt)
+TEST_FOR_TYPE_NAMED_OBJ(ATE, AdaptiveTimeSteppingSimple, serializationTestObjectSimple)
```
Comment (bska) [inline tests/test_RestartSerialization.cpp:119]: Tiny nit, but `ATE` probably isn't a good name anymore once we remove the `Ebos` part of the class name.  How about `ATS` instead?

**Review by bska — COMMENTED** [review]
Looks good except for one small naming problem.

File: tests/test_RestartSerialization.cpp (hunk 1 above)
Comment (akva2) [reply tests/test_RestartSerialization.cpp]: okay

**Comment by akva2** [comment]
jenkins build this please

**Review by bska — APPROVED** [review]
Very good.  I'll merge into master.

**Pull request merged**